app/javascript/gobierto_investments/index.js

```diff
@@ -1,3 +1,5 @@
+import "../../assets/stylesheets/module-investments.scss"
```

Comment:
Here we set the reference to the module CSS; in that way, webpack will add this file to the proper chunk.